### PR TITLE
ADEN-11235 fix CLS on sticky TLB

### PR DESCRIPTION
**Pull request by jakubbanas**

### CODE REVIEW

File: platforms/ucp-desktop/styles.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
 			@include bfaa-slot(402, $global-navigation-width);
 			@include sticky-tlb-slot($global-navigation-width);
 
-			margin-left: $global-navigation-width;
+			margin-left: $global-navigation-width !important;
```
Comment (jakubbanas) [inline platforms/ucp-desktop/styles.scss:31]: 🙈

**Comment by jakubbanas** [comment]
About the importants: Rotation Media is messing with our CSS, changing TLB width.
We could ask them to stop doing that, and then we could remove those importants.

File: platforms/shared/styles/uap/_bfaa-slot.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	left: 0;
 	top: 0;
 	position: fixed;
-	width: calc(100% - #{$uap-margin});
+	width: calc(100% - #{$uap-margin}) !important;
```
Comment (dmnsobczak) [inline platforms/shared/styles/uap/_bfaa-slot.scss:9]: Comment: it has to be this way to override DurationMedia's `!important` :<

**Review by dmnsobczak — APPROVED** [review]

File: platforms/ucp-desktop/setup/dynamic-slots/ucp-desktop-dynamic-slots.setup.ts [hunk 3]
```diff
@@ -205,8 +205,13 @@ export class UcpDesktopDynamicSlotsSetup implements DiProcess {
 			}
 
 			if (context.get('templates.stickyTlb.forced')) {
-				context.push('slots.top_leaderboard.defaultTemplates', 'stickyTlb');
-				context.push('slots.hivi_leaderboard.defaultTemplates', 'stickyTlb');
+				communicationService.action$.pipe(ofType(uapLoadStatus), take(1)).subscribe((action) => {
```
Comment (nandy-andy) [inline platforms/ucp-desktop/setup/dynamic-slots/ucp-desktop-dynamic-slots.setup.ts:208]: Oh, this fixes the issue with weird looking UAP 👌

File: platforms/ucp-desktop/styles.scss (hunk 1 above)
Comment (nandy-andy) [reply platforms/ucp-desktop/styles.scss:31]: Is it also because of DurationMedia? :(

Comment (jakubbanas) [reply platforms/ucp-desktop/styles.scss:31]: Yes, it is. I created a [follow-up ticket](https://fandom.atlassian.net/browse/ADEN-11428) to fix it.

File: platforms/shared/styles/uap/_bfaa-slot.scss (hunk 2 above)
Comment (preludek) [reply platforms/shared/styles/uap/_bfaa-slot.scss:9]: !important war xd

File: platforms/shared/templates/handlers/sticky-tlb/sticky-tlb-blocking-handler.ts [hunk 4]
```diff
@@ -17,16 +21,27 @@ export class StickyTlbBlockingHandler implements TemplateStateHandler {
...
-			this.adSlot.emitEvent(universalAdPackage.SLOT_STICKINESS_DISABLED);
-			this.logger(
-				`Template 'stickyTlb' could not be applied for Line item ID ${this.adSlot.lineItemId}`,
-			);
+		const isUap = await this.isUAP();
+		if (isUap) {
+			this.blockStickyTLB('UAP');
+			return;
 		}
+		if (!this.isStickyTlbForced() && !this.isLineAndGeo()) {
```
Comment (dmnsobczak) [inline platforms/shared/templates/handlers/sticky-tlb/sticky-tlb-blocking-handler.ts:29]: "and" of two negations are always hard to read :D

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**